packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/NetworkInfoBuilder.ts

```diff
@@ -62,4 +65,28 @@ export class NetworkInfoBuilder {
     );
     return result.rows[0];
   }
+
+  public async queryMultiAsset(policyId: Cardano.PolicyId, name: Cardano.AssetName) {
```

Comment:
Whoops, approved it a bit soon :roll_eyes:

I've just noticed this, the queries, and types should be in new files:
- `src/Asset/AssetBuilder.ts`
- `src/Asset/queries.ts`
- `src/Asset/types.ts`